exercises/robot-simulator/example.js

```diff
@@ -1,4 +1,4 @@
-var Robot = (function () {
+var robotOne  = (function () {
```

Comment:
Can you remove this function wrapping? This should remove the need for this extra variable - the name `robotOne` seems a little odd. Should look something like this:

```js
'use strict';

var VALID_DIRECTIONS = ['north', 'east', 'south', 'west'];
var INSTRUCTION_KEYS = {
  A: 'advance',
  L: 'turnLeft',
  R: 'turnRight'
};

function Robot() {
  this.coordinates = [0, 0];
  this.bearing = 'north';
}

Robot.prototype.at = function (x, y) {
  this.coordinates = [x, y];
};
....
....
```

If you are confused, here is how the entire file will look like: https://gist.github.com/tejasbubane/ef04a2d3d5f88858f6f2ddd1c8b6c92b plus the default condition change which you have in this PR.